src/types/H5P/H5PField.ts

```diff
@@ -18,6 +18,7 @@ type H5PTextFieldWidgetExtension =
   | H5PFieldWidgetExtension
   | {
       widget?: "html";
+      tags: Array<H5PTextTags>;
```

Comment:
Should it be optional if widget is optional? 

```suggestion
      tags?: Array<H5PTextTags>;
```

Reply:
No, I don't think so? I think we should use `tags: []` if we don't want to allow any tags